public/docs/dart/latest/guide/template-syntax.jade

```diff
@@ -1,407 +1,125 @@
-+includeShared('{ts}', 'event-binding-8')
-
-+includeShared('{ts}', 'ngModel-1')
-+makeExample('template-syntax/dart/lib/app_component.html', 'NgModel-1')(format=".")
-+includeShared('{ts}', 'ngModel-2')
-+makeExample('template-syntax/dart/lib/app_component.html', 'NgModel-2')(format=".")
-+includeShared('{ts}', 'ngModel-3')
-+makeExample('template-syntax/dart/lib/app_component.html', 'without-NgModel')(format=".")
-+includeShared('{ts}', 'ngModel-4')
-+makeExample('template-syntax/dart/lib/app_component.html', 'NgModel-3')(format=".")
-+includeShared('{ts}', 'ngModel-5')
-+makeExample('template-syntax/dart/lib/app_component.html', 'NgModel-1')(format=".")
-+includeShared('{ts}', 'ngModel-6')
-+makeExample('template-syntax/dart/lib/app_component.html', 'NgModel-4')(format=".")
-+includeShared('{ts}', 'ngModel-7')
-
-+includeShared('{ts}', 'directives-1')
-+makeExample('template-syntax/dart/lib/app_component.html', 'event-binding-1')(format=".")
-+includeShared('{ts}', 'directives-2')
-+includeShared('{ts}', 'directives-ngClass-1')
-+makeExample('template-syntax/dart/lib/app_component.html', 'class-binding-3a')(format=".")
-+includeShared('{ts}', 'directives-ngClass-2')
-+makeExample('template-syntax/dart/lib/app_component.dart', 'setClasses')(format=".")
-// PENDING: Add "Dart difference" for returning maps in Dart? for omitting unnecessary `this.`s?
-+includeShared('{ts}', 'directives-ngClass-3')
-+makeExample('template-syntax/dart/lib/app_component.html', 'NgClass-1')(format=".")
-+includeShared('{ts}', 'directives-ngStyle-1')
-+makeExample('template-syntax/dart/lib/app_component.html', 'NgStyle-1')(format=".")
-+includeShared('{ts}', 'directives-ngStyle-2')
-+makeExample('template-syntax/dart/lib/app_component.dart', 'setStyles')(format=".")
-+includeShared('{ts}', 'directives-ngStyle-3')
-+makeExample('template-syntax/dart/lib/app_component.html', 'NgStyle-2')(format=".")
-//
-  +includeShared('{ts}', 'directives-ngIf-1')
-<a id="ngIf"></a>
-.l-main-section
-:marked
-  ### NgIf
-  We can add an element subtree (an element and its children) to the DOM  by binding an `NgIf` directive to a `true` expression.
-+makeExample('template-syntax/dart/lib/app_component.html', 'NgIf-1')(format=".")
-+includeShared('{ts}', 'directives-ngIf-2')
-//
-  +includeShared('{ts}', 'directives-ngIf-3')
-:marked
-  Binding to a false expression removes the element subtree from the DOM.
-+makeExample('template-syntax/dart/lib/app_component.html', 'NgIf-2')(format=".")
-.callout.is-helpful
-  header Dart difference: No truthy values
+    Perhaps more surprising, template expressions can’t refer to static
+    properties, nor to top-level variables or functions, such as `window` or
+    `document` from `dart:html`. They can’t directly call `print` or functions
+    imported from `dart:math`. They are restricted to referencing members of
+    the expression context.
+
+block array-is-list-in-Dart
+  .callout.is-helpful
+    header Dart difference: Arrays are lists
```

Comment:
> Or we could just use "list" everywhere

Agreed.

> We'd still want to call out that difference somewhere (perhaps in this chapter)

Agreed. I don't think that it should be nestled in a central chapter of the basic dev. guide. So I'll just cut the callout out and save it for inclusion elsewhere.